source/Octopus.Client/OctopusRepository.cs

```diff
@@ -221,12 +239,12 @@ SpaceRootResource LoadSpaceRootDocumentInner()
                 () =>
                 {
                     var defaultSpace = TryGetDefaultSpace();
-                    return defaultSpace != null ? LoadSpaceRootResourceFor(defaultSpace.Id) : null;
+                    return defaultSpace != null ? LoadSpaceRootResourceFor(defaultSpace) : null;
                 });
 
-            SpaceRootResource LoadSpaceRootResourceFor(string spaceId)
+            SpaceRootResource LoadSpaceRootResourceFor(SpaceResource space)
             {
-                return Client.Get<SpaceRootResource>(loadRootResource.Value.Link("SpaceHome"), new {spaceId});
+                return Client.Get<SpaceRootResource>(loadRootResource.Value.Link("SpaceHome"), new {space.Id});
```

Comment:
We can do this a bit nicer now that we have the `SpaceResource`
```suggestion
                return Client.Get<SpaceRootResource>(space.Link("SpaceHome"));
```

Reply:
done